api/v1alpha1/storagebasedremediationconfig_types.go

```
@@ -90,6 +93,17 @@ type StorageBasedRemediationConfigSpec struct {
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// SBRTimeoutSeconds configures the base timing for failure detection.
+	// The heartbeat is sent every (sbrTimeoutSeconds/2) seconds.
+	// A node is considered unhealthy after maxConsecutiveFailures missed heartbeats.
+	// The operator also uses (sbrTimeoutSeconds/6) for update and peer-check intervals (~3 scans per heartbeat) so shared-storage jitter is less likely to look like a missed heartbeat.
```

golangci-lint (2.12.2)

[error] The line is 182 characters long, which exceeds the maximum of 120 characters.

(lll)